src/charm.py

```diff
@@ -630,12 +596,8 @@ def _downstream_grafana_dashboard_relation_broken(self, _):
     def _prometheus_target_relation_joined(self, _):
         self._stored.have_targets = True
 
-    def _prometheus_target_relation_changed(self, event: RelationChangedEvent):
-        self._handle_prometheus_alert_rule_files(RULES_DIR, event.app.name)
```

Comment:
Similar to the question above: this like here suggests that at some point we thought that prometheus-target relation comes with rule files, but in this PR we seem to remove all references to prometheus-target.